src/topology/algebra/module.lean

```diff
@@ -749,7 +749,7 @@ variables {R S : Type*} [ring R] [ring S] [topological_space S]
   [module S M₃] [smul_comm_class R S M₃] [has_continuous_smul S M₃]
 
 instance : has_scalar S (M →L[R] M₃) :=
-⟨λ c f, ⟨c • f, continuous_const.smul f.2⟩⟩
+⟨λ c f, ⟨c • f, (continuous_const.smul f.2 : continuous (λ x, c • f x))⟩⟩
```

Comment:
This is an example of one of the proofs broken by changing how `linear_map.has_scalar` unfolds.